### PR TITLE
fix: report bug button AR-2650

### PR DESCRIPTION
**Pull request by tmspzz**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/AR-2650" title="AR-2650" target="_blank">BugAR-2650</a>  Report Bug button not working anymore
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
    - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
    - [x] contains a reference JIRA issue number like `SQPIT-764`
    - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
    - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

Report bug button does nothing, therefore users can't report bugs. A software development 101 classic.

### Solutions

Provide a valid file path to the intent that attaches logs

#### How to Test

Tap Report Bug -> See intent request popping up

----
#### PR Post Submission Checklist for internal contributors (Optional)

- [x] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

- [x] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Comment by AndroidBob** [comment]
Build [1](https://10.10.124.134/job/wire-android-reloaded/job/PR-1108/1/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(4739549)(4739549)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(4739549)(4739549)-dev-release.apk)

File: app/src/main/kotlin/com/wire/android/ui/home/HomeViewModel.kt [hunk 1]
```diff
@@ -113,7 +114,7 @@ class HomeViewModel @Inject constructor(
 data class HomeState(
     val avatarAsset: UserAvatarAsset? = null,
     val status: UserAvailabilityStatus = UserAvailabilityStatus.NONE,
-    val logFilePath: String = String.EMPTY
+    val logFilePath: String
```
Comment (gongracr) [inline app/src/main/kotlin/com/wire/android/ui/home/HomeViewModel.kt:117]: 😅 🙈

File: app/src/main/kotlin/com/wire/android/ui/home/HomeDrawer.kt [hunk 2]
```diff
@@ -137,12 +137,12 @@ fun HomeDrawer(
...
-                val logsUris = context.getUrisOfFilesInDirectory(dir)
-                val intent = context.multipleFileSharingIntent(logsUris)
-                intent.putExtra(Intent.EXTRA_EMAIL, arrayOf("wire-newandroid@wearezeta.zendesk.com"))
-                intent.putExtra(Intent.EXTRA_SUBJECT, "Bug Report - Wire Beta")
-                intent.putExtra(Intent.EXTRA_TEXT, EmailComposer.reportBugEmailTemplate(context.getDeviceId()?.sha256()))
-                intent.type = "message/rfc822"
+                    val logsUris = context.getUrisOfFilesInDirectory(dir)
+                    val intent = context.multipleFileSharingIntent(logsUris)
+                    intent.putExtra(Intent.EXTRA_EMAIL, arrayOf("wire-newandroid@wearezeta.zendesk.com"))
+                    intent.putExtra(Intent.EXTRA_SUBJECT, "Bug Report - Wire Beta")
```
Comment (gongracr) [inline app/src/main/kotlin/com/wire/android/ui/home/HomeDrawer.kt:143]: Do we want to localise this for also German? or is it rather universal tech language 😅 ?

**Review by gongracr — APPROVED** [review]
lgtm 📩

**Pull request merged**